src/PostgREST/Auth.hs

```diff
@@ -23,6 +23,7 @@ import           Control.Lens
 import           Data.Aeson              (Value (..), decode, toJSON)
 import qualified Data.ByteString.Lazy    as BL
 import qualified Data.HashMap.Strict     as M
+import           Data.Maybe (fromJust)
```

Comment:
This is nitpicky, but how about indenting fromJust to match the rightmost column of the imports?

Reply:
Done. I would suggest to even go further and to make it 'work' with stylish-haskell (which would involve separate the protolude line). As a note, the protolude line is usually at the bottom instead of the top of the import list IIRC.